tests/test_io/test_h5simple/test_h5simple.py

```
@@ -39,3 +46,39 @@ def test_dims_in_attrs(self, h5simple_with_dim_attrs_path):
+
+
+class TestH5SimpleInternalHelpers:
+    """Direct tests for helper branches that still support PyTables fixtures."""
+
+    def test_get_root_attrs_supports_pytables(self, tmp_path):
+        """PyTables handles should expose root attrs through the helper."""
+        path = tmp_path / "root_attrs.h5"
+        with tables.open_file(path, "w") as h5:
+            h5.root._v_attrs["dims"] = "distance,time"
+            attrs = _get_root_attrs(h5)
+            assert getattr(attrs, "dims") == "distance,time"
```

Ruff (0.15.7)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)